c-dependencies/js-compute-runtime/builtins/config-store.cpp

```diff
@@ -0,0 +1,66 @@
+#include "config-store.h"
```

Comment:
@elliottt - this and config-store.h are nearly identical copies of the already existing dictionary.cpp and dictionary.h files but with the builtin renamed to ConfigStore